registration/forms.py

```diff
@@ -247,7 +247,6 @@ def create_user(self, application, commit=True):
         user = User(
             username=self.cleaned_data['username'],
             email=self.cleaned_data['email'],
-            is_active=False,
```

Comment:
We can simply check the email validation status when updating the status to
'approved'. This could be done as a 2-line method on the model.